wls-vorfaelleundvorkommnisse-service/src/test/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/service/ereignis/EreignisServiceSecurityTest.java

```diff
@@ -0,0 +1,143 @@
+
+    @Autowired
+    EreignisRepository ereignisRepository;
+
+    @BeforeEach
+    void setup() {
+        SecurityContextHolder.clearContext();
+    }
+
+    @Nested
+    class GetEreignis {
+
+        @Test
+        void should_not_throw_exception_when_given_all_authorities() {
+            val wahlbezirkID = "wahlbezirkID";
+
+            SecurityUtils.runWith(Authorities.ALL_AUTHORITIES_GET_EREIGNISSE);
+            Mockito.when(bezirkIDPermissionEvaluator.tokenUserBezirkIdMatches(Mockito.eq(wahlbezirkID), Mockito.any())).thenReturn(true);
+
+            Assertions.assertThatNoException().isThrownBy(() -> unitUnderTest.getEreignis(wahlbezirkID));
+        }
+
+        @Test
+        void should_throw_AccessDeniedException_when_bezirkIDPermissionEvaluator_missing() {
+            val wahlbezirkID = "wahlbezirkID";
+
+            SecurityUtils.runWith(Authorities.ALL_AUTHORITIES_GET_EREIGNISSE);
+            Mockito.when(bezirkIDPermissionEvaluator.tokenUserBezirkIdMatches(Mockito.eq(wahlbezirkID), Mockito.any())).thenReturn(false);
+
+            Assertions.assertThatThrownBy(() -> unitUnderTest.getEreignis(wahlbezirkID)).isInstanceOf(AccessDeniedException.class);
+        }
+
+        @ParameterizedTest(name = "{index} - {1} missing")
+        @MethodSource("getMissingAuthoritiesVariations")
+        void should_throw_AccessDeniedException_when_any_authority_missing(final ArgumentsAccessor argumentsAccessor) {
+            SecurityUtils.runWith(argumentsAccessor.get(0, String[].class));
+
+            val wahlbezirkID = "wahlbezirkID";
+            Mockito.when(bezirkIDPermissionEvaluator.tokenUserBezirkIdMatches(Mockito.eq(wahlbezirkID), Mockito.any())).thenReturn(true);
+
+            Assertions.assertThatThrownBy(() -> unitUnderTest.getEreignis(wahlbezirkID)).isInstanceOf(AccessDeniedException.class);
+        }
+
+        private static Stream<Arguments> getMissingAuthoritiesVariations() {
+            return SecurityUtils.buildArgumentsForMissingAuthoritiesVariations(Authorities.ALL_AUTHORITIES_GET_EREIGNISSE);
+        }
+    }
+
+    @Nested
+    class SetEreignis {
+
+        @Test
+        void should_not_throw_exception_when_given_all_authorities() {
+            val wahlbezirkID = "wahlbezirkID";
+            List<EreignisModel> ereignisModelList = new ArrayList<>();
+            ereignisModelList.add(TestdataFactory.createEreignisModelWithData());
+            val ereignisseWriteModel = TestdataFactory.createEreignisseWriteModelWithData(wahlbezirkID, ereignisModelList);
+
+            SecurityUtils.runWith(Authorities.ALL_AUTHORITIES_SET_EREIGNISSE);
+            Mockito.when(bezirkIDPermissionEvaluator.tokenUserBezirkIdMatches(Mockito.eq(wahlbezirkID), Mockito.any())).thenReturn(true);
+
+            Assertions.assertThatNoException().isThrownBy(() -> unitUnderTest.postEreignis(ereignisseWriteModel));
+        }
+
+        @Test
+        void should_throw_AccessDeniedException_when_bezirkIDPermissionEvaluator_missing() {
```

Comment:
analog zu `getEreignisse`

Reply:
siehe commit [50cad96](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/50cad968e13e70c97c398f5338f1d8d08882e7df)